<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCharacterPad</class>
 <widget class="QWidget" name="CCharacterPad">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>1</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>CCharacterPad</string>
  </property>
  <property name="styleSheet">
   <string notr="true">CCharacterPad 
{
    background-color:qlineargradient(spread:pad, x1:0.460227, y1:0, x2:0.443091, y2:0.898, stop:0 rgba(74,79,80,255), stop:1 rgba(65,69,65,255));
}

QPushButton
{
    color:#354346;
    background-color:qlineargradient(spread:pad, x1:0.460227, y1:0, x2:0.443091, y2:0.898, stop:0 rgba(245,245,245,255), stop:1 rgba(185,187,192,255));
    border-radius:5px;
    font-weight: bold;
    font-size: 14pt;
    text-align: center;
    padding: -2;
    margin: 0;
}
QPushButton:focus
{
    color:#ececec;
    background-color:qlineargradient(spread:pad, x1:0.460227, y1:0, x2:0.443091, y2:0.898, stop:0 rgba(94,132,138,255), stop:1 rgba(161,176,200,255));
}
QPushButton:pressed
{
    color:#e6e7e2;
    background-color:qlineargradient(spread:pad, x1:0.460227, y1:0, x2:0.443091, y2:0.898, stop:0 rgba(68,68,68,255), stop:1 rgba(37,37,37,255));
}
QPushButton#pushButton_Enter:focus,
QPushButton#pushButton_Cancel:focus,
QPushButton#pushButton_Del:focus,
QPushButton#pushButton_CHAR:focus,
QPushButton#pushButton_NUM:focus
{
    color:#ececec;
    background-color:qlineargradient(spread:pad, x1:0.460227, y1:0, x2:0.443091, y2:0.898, stop:0 rgba(94,132,138,255), stop:1 rgba(161,176,200,255));
}
QPushButton#pushButton_Enter
{
    color:white;
    background-color:green;
}
QPushButton#pushButton_Cancel
{
    color:white;
    background-color:#f8a119;
}
QPushButton#pushButton_Del
{
    color:white;
    background-color:red;
}
QPushButton#pushButton_Del:pressed
{
    color:#e6e7e2;
    background-color:qlineargradient(spread:pad, x1:0.460227, y1:0, x2:0.443091, y2:0.898, stop:0 rgba(68,68,68,255), stop:1 rgba(37,37,37,255));
} QPushButton#pushButton_NUM,QPushButton#pushButton_CHAR
{
    color:black;
    background-color:#a4c500;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_Q">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>2</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_W">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>2</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>w</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_E">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>2</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>e</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_R">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>2</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>r</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_T">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>2</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>t</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Y">
   <property name="geometry">
    <rect>
     <x>403</x>
     <y>2</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_U">
   <property name="geometry">
    <rect>
     <x>483</x>
     <y>2</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>u</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_I">
   <property name="geometry">
    <rect>
     <x>563</x>
     <y>2</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>i</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_O">
   <property name="geometry">
    <rect>
     <x>643</x>
     <y>2</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>o</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_P">
   <property name="geometry">
    <rect>
     <x>723</x>
     <y>2</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>p</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Del">
   <property name="geometry">
    <rect>
     <x>723</x>
     <y>52</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_L">
   <property name="geometry">
    <rect>
     <x>643</x>
     <y>52</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_J">
   <property name="geometry">
    <rect>
     <x>483</x>
     <y>52</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>j</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_A">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>52</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>a</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_S">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>52</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>s</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_F">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>52</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>f</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_H">
   <property name="geometry">
    <rect>
     <x>403</x>
     <y>52</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>h</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_K">
   <property name="geometry">
    <rect>
     <x>563</x>
     <y>52</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>k</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_D">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>52</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>d</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_G">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>52</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>g</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_V">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>102</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_N">
   <property name="geometry">
    <rect>
     <x>483</x>
     <y>102</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>n</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_M">
   <property name="geometry">
    <rect>
     <x>563</x>
     <y>102</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_X">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>102</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_B">
   <property name="geometry">
    <rect>
     <x>403</x>
     <y>102</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>b</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Z">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>102</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_C">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>102</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>c</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>643</x>
     <y>102</y>
     <width>155</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Caps">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>102</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>caps</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_NUM">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>153</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>NUM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Up">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>153</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>上</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Down">
   <property name="geometry">
    <rect>
     <x>403</x>
     <y>153</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>下</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Right">
   <property name="geometry">
    <rect>
     <x>563</x>
     <y>153</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>右</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Left">
   <property name="geometry">
    <rect>
     <x>483</x>
     <y>153</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>左</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Space">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>153</y>
     <width>235</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Enter">
   <property name="geometry">
    <rect>
     <x>643</x>
     <y>153</y>
     <width>155</width>
     <height>45</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
